compiler/rustc_passes/src/hir_stats.rs

```diff
@@ -560,13 +560,14 @@ impl<'v> ast_visit::Visitor<'v> for StatCollector<'v> {
     }
 
     fn visit_expr(&mut self, e: &'v ast::Expr) {
+        #[rustfmt::skip]
```

Comment:
Nit: what happens if you remove this? It's probably better to just keep the defaults and use whatever rustfmt produces.

Reply:
The original `rustfmt` output can be seen in https://github.com/rust-lang/rust/pull/103812/commits/a8ddb5656370c58f81360255a9efc68e102a4abd, others suggested I should leave it as the original format is a little easier to read

Reply:
Ok, that's pretty bad, let's keep the `#[rustfmt::skip]` then.